README.md

```diff
@@ -34,15 +35,52 @@ The assisted-service UI is available on port 8080.
 Node0
 -------
 
-To run the assisted service only on a pre-determined node a.k.a. node0, currently we have hardcoded a static IP 192.168.122.2. 
-A systemd service named `node-zero.service` looks for the static IP and if the current node's IP matches with it then only the `assisted-service.service` systemd service is startred. The `assisted-service.service` is responsible for running the assisted service.
+To run the assisted service only on a pre-determined node a.k.a. node0, the node0 IP must be defined in nmstateconfig.yaml with a mac address that matches that node, for example:
 
-To set the static ip in libvirt:
+```
+apiVersion: agent-install.openshift.io/v1beta1
+kind: NMStateConfig
+metadata:
+  name: mynmstateconfig
+  namespace: spoke-cluster
+  labels:
+    cluster0-nmstate-label-name: cluster0-nmstate-label-value
+spec:
+  config:
+    interfaces:
+      - name: eth0
+        type: ethernet
+        state: up
+        mac-address: 52:54:01:aa:aa:a1
+        ipv4:
+          enabled: true
+          address:
+            - ip: 192.168.122.2
+              prefix-length: 24
+          dhcp: false
+    dns-resolver:
+      config:
+        server:
+          - 192.168.122.1
+    routes:
+      config:
+        - destination: 0.0.0.0/0
+          next-hop-address: 192.168.122.1
+          next-hop-interface: eth0
+          table-id: 254
+  interfaces:
+    - name: "eth0"
+      macAddress: "52:54:01:aa:aa:a1"
+```
+
+A systemd service named `node-zero.service` looks for the static IP and if the current node's IP matches with it then only the `assisted-service.service` systemd service is started. The `assisted-service.service` is responsible for running the assisted service.
+
+In order to supply a hostname to this node a DNS entry must be added:
 1. Edit the default network
 ```
 virsh net-edit default
```

Comment:
I'd keep the current README file as much as possible agnostic and independent by the specific environment setup (we're currently using vms with libvirt, but that's our choice for our environment)/ I think it's very useful to describe/report the higher level requirements , but will leave out the specific technical implementation that may depend on particular user' choices